platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -157,6 +158,12 @@ def getRepositoryPassword() {
             (hasProperty('NEXUS_PASSWORD') ? NEXUS_PASSWORD : "")
 }
 
+def getGitRevision() {
```

Comment:
Yes, I saw that. But this way we can initialise the string statically instead of passing the revision through JNI in the constructor of each http request. Minor optimisation, but it makes things a little cleaner as well.